packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -54,8 +54,16 @@ export interface OrderStatusExtensionTargets {
     OrderStatusApi<'customer-account.order-status.cart-line-list.render-after'>,
     AnyComponent
   >;
-  'customer-account.order-status.delivery-details.render-after': RenderExtension<
-    OrderStatusApi<'customer-account.order-status.delivery-details.render-after'>,
+  'customer-account.order-status.return-details.render-after': RenderExtension<
```

Comment:
I noticed the figma has order-details instead of order-status. Do we want to use order-status or its a typo in the figma?